onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol

```diff
@@ -622,29 +623,39 @@ contract CartesiDAppTest is TestBase {
         vouchers[index] = Voucher(destination, payload);
     }
 
-    function checkInputIndex(uint256 inputIndex) internal view {
+    // IIWE stands for InputIndexWithinEpoch
+    function checkIIWE(uint256 inputIndexWithinEpoch) internal view {
```

Comment:
This might save some characters, but we lose clarity by abbreviating `InputIndexWithinEpoch` to `IIWE`.
I would, therefore, avoid abbreviating it.
This applies to `IIWEStr` as well.

Reply:
Yeah. It's always a tradeoff between being clear and concise. Since they don't appear often, we could use the lengthy form